### PR TITLE
add CPGRL doc section

### PR DESCRIPTION
**Pull request by drammock**
this new section will be referenced in our CoC, once https://github.com/mne-tools/.github/pull/1/ is merged.

### CODE REVIEW

**Comment by drammock** [comment]
cc @britta-wstnr @hoechenberger @sappelhoff @wmvanvliet who all suggested/agreed that there should be more diversity among the CPGRLs.  Suggestions (and self-nominations) are welcome.

File: doc/overview/people.rst [hunk 1]
```diff
@@ -33,6 +33,18 @@ Steering Council
 * `Robert Luke`_
 * `Stefan Appelhoff`_
 
+
+.. _governance-cpgrl:
+
+Community Participation Guidelines Response Leads
+-------------------------------------------------
+
+See our `Community Participation Guidelines<https://github.com/mne-tools/.github/blob/main/CODE_OF_CONDUCT.md>`_ for explanation. Current Response Leads are:
```
Comment (drammock) [inline doc/overview/people.rst:42]:
```suggestion
See our `Community Participation Guidelines<https://github.com/mne-tools/.github/blob/main/CODE_OF_CONDUCT.md>`__ for explanation. Current Response Leads are:
```

File: doc/overview/people.rst [hunk 2]
```diff
@@ -33,6 +33,18 @@ Steering Council
 * `Robert Luke`_
 * `Stefan Appelhoff`_
 
+
+.. _governance-cpgrl:
+
+Community Participation Guidelines Response Leads
+-------------------------------------------------
+
+See our `Community Participation Guidelines<https://github.com/mne-tools/.github/blob/main/CODE_OF_CONDUCT.md>`__ for explanation. Current Response Leads are:
```
Comment (drammock) [inline doc/overview/people.rst:42]:
```suggestion
See our `Community Participation Guidelines <https://github.com/mne-tools/.github/blob/main/CODE_OF_CONDUCT.md>`__ for explanation. Current Response Leads are:
```

**Comment by larsoner** [comment]
You probably need a merge with `main` to get the YAML to show up and actually run

**Comment by larsoner** [comment]
... nope, your branch isn't that out of date. I'll look deeper

**Comment by larsoner** [comment]
Weird, it's queued now with a merge with `main`. Maybe some transient GH actions trigger bug?

**Pull request merged**